CMakeLists.txt

```diff
@@ -228,18 +230,19 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 if(PKG_CONFIG_FOUND)
     pkg_check_modules(ZLIB zlib)
```

Comment:
BTW this is not marked `REQUIRED`.

Reply:
That is intentional. For me, at least, it isn't a requirement. @diizzyy can comment on whether it is a requirement or a nice to have.

I've always presumed that Cmake does the same thing via `findpackage`. Unlike meson which (afaik) relies on pkgconf more (at least on Unix).